.github/workflows/php.yml

```diff
@@ -1,41 +1,122 @@
-name: Tests
+name: "Integrate"
```

Comment:
I won't check this file thoroughly. I simply assume that you re-used the file from LycheeOrg/Lychee without modifications.

Reply:
pretty much.